### ISSUE TITLE
Certain FP16 traced pytorch models with batchnorm no longer work with r21.07

### ISSUE THREAD

**Issue by jamt9000** [triton-inference-server/server#3191]
**Description**

Certain pytorch traced models now give errors like

```
in ensemble 'jasper-ts-trace-ensemble', PyTorch execute failure: MALFORMED INPUT: bad dtype in CompareSelect
```

In this case with https://ngc.nvidia.com/catalog/models/nvidia:jasper_pyt_torchscript_fp16_amp/version and [fp16/jasper-ts-trace-ensemble/config.pbtxt](https://github.com/NVIDIA/DeepLearningExamples/blob/d788e8d4968e72c722c5148a50a7d4692f6e7bd3/PyTorch/SpeechRecognition/Jasper/triton/model_repo_configs/fp16/jasper-ts-trace-ensemble/config.pbtxt)

This is likely to be caused by this pytorch batchnorm/fusing bug https://github.com/pytorch/pytorch/issues/61382 https://github.com/pytorch/pytorch/issues/61336 but I'm opening an issue here since it's likely to affect triton fp16 inference usecases and affects nvidia's released models. Perhaps it should be included as a known issue at least.

**Triton Information**

r21.06 or r21.07 prebuilt

**To Reproduce**

For example the nvidia Jasper model, as in https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechRecognition/Jasper/triton#inference-pipeline-in-triton-inference-server

**Expected behavior**

It should work as before with no error

**Comment by kommons** [comment]
Upstream Pytorch issues are now fixed and will be included in our 21.08 release. Please try again with our 12.08 and re-open if you are still seeing failures.
